app/components/Sidebar/components/ArchiveLink.tsx

```diff
@@ -61,7 +61,6 @@ function ArchiveLink() {
           icon={<ArchiveIcon open={isDocumentDropping} />}
           exact={false}
           label={t("Archive")}
-          active={documents.active?.isArchived && !documents.active?.isDeleted}
```

Comment:
Made this choice to prevent scenarios like 👇

<img width="241" alt="Screenshot 2024-09-11 at 3 54 15 PM" src="https://github.com/user-attachments/assets/ef3de04a-e83b-4133-8d53-48e3d5318321">

Doesn't look good!

That being said, same behavior holds for the current collection structure in sidebar. If you make any doc under it _active_, the highlight is stolen from the collection.

Reply:
Yes, it should match collection->document highlight behavior 👍